test/persistence/kafka-sources/exactly-once-sink-after.td

```diff
@@ -0,0 +1,41 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+$ set keyschema={
+    "type": "record",
+    "name": "Key",
+    "fields": [
+        {"name": "f1", "type": "long"}
+    ]
+  }
+
+$ set schema={
+        "type" : "record",
+        "name" : "test",
+        "fields" : [
+            {"name":"f2", "type":"long"}
+        ]
+    }
+
+$ kafka-ingest format=avro topic=exactly-once key-format=avro key-schema=${keyschema} schema=${schema} publish=true repeat=2 start-iteration=40
+{"f1": ${kafka-ingest.iteration}} {"f2": ${kafka-ingest.iteration}}
+
+
+> SELECT COUNT(*) = 10 FROM exactly_once;
+true
+
+# We expect just the 2 new messages, having consumed the older ones in xactly-once-sink-before.td
```

Comment:
This comment was a bit misleading for me, because we also read the 20,21,30,31 messages that we write in the "before" test.